src/components/Votes.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+
+function Votes({ voteData }) {
+  const [upVote, setUpVote] = useState(0);
+  return (
+    <div>
+      <p>Votes: {voteData + upVote}</p>
+      <button
+        onClick={() => {
+          setUpVote((curr) => (curr += 1));
+          console.log(upVote);
+        }}
+      >
```

Comment:
Nice one splitting out into its own component.
It updates the votes on the page, but not in the database. We need to also do the api request when we click the button. 
You'll also want to make sure you think about what happens if the api request fails, and make sure that the number on the page, is still reflective of the database